src/components/screens/sprint-view-screen.tsx

```diff
@@ -172,19 +138,19 @@ const SprintViewScreen = () => {
     <>
       {loading ? (
         <Card sx={{ p: "25%", display: "flex", justifyContent: "center" }}>
-          <Box sx={{textAlign: 'center'}} >
+          <Box sx={{ textAlign: "center" }} >
             <Typography>{strings.placeHolder.pleaseWait}</Typography>
-            <CircularProgress sx={{ scale: "150%", mt: "5%", mb: "5%"}} />
+            <CircularProgress sx={{ scale: "150%", mt: "5%", mb: "5%" }} />
           </Box>
         </Card>
       ) : (
         <>
           <FormControlLabel control={<Switch checked={myTasks}/>} label={strings.sprint.showMyTasks}  onClick={() =>  handleOnClickTask()}/>  
```

Comment:
Loads of components within this render could have their props indented on a new line. Remember as a general rule if a component has more 3 or more props each should be indented on a new line- this massively improves code clarity